src/commands/utility/brainfuck.js

```diff
@@ -0,0 +1,43 @@
+const { Command } = require('../../')
+const Brainfuck = require('brainfuck-node')
+const brainfuck = new Brainfuck()
+
+module.exports = class BrainfuckCommand extends Command {
+  constructor (client) {
+    super({
+      name: 'brainfuck',
+      category: 'utility',
+      aliases: ['brainf', 'brf'],
+      parameters: [
+        {
+          type: 'string',
+          required: true,
+          missingError: 'commands:brainfuck.noCode'
+        },
+        {
+          type: 'string',
+          full: true,
+          required: false
+        }
+      ]
+    }, client)
+  }
+  constructOutputMsg (input, t) {
+    const output = `\`\`\`${input.output}\`\`\``
+    return `**${t('commands:brainfuck.steps')}:** ${input.steps}\n**${t('commands:brainfuck.time')}:** ${input.time}ms\n**${t('commands:brainfuck.output')}:** ${output}`
+  }
+  run ({ t, channel }, code, input) {
```

Comment:
Add a line between functions please
```suggestion
  }
  
  constructOutputMsg (input, t) {
    const output = `\`\`\`${input.output}\`\`\``
    return `**${t('commands:brainfuck.steps')}:** ${input.steps}\n**${t('commands:brainfuck.time')}:** ${input.time}ms\n**${t('commands:brainfuck.output')}:** ${output}`
  }
  
  run ({ t, channel }, code, input) {
```